### ISSUE TITLE
Duik Connectors

### ISSUE THREAD

**Issue by MarkD76** [airbnb/lottie-web#1712]
I was trying out Connectors in Duik Basel and wanted to see if animation could be exported as JSON with Bodymovin.
when I render all I see is a blanks screen, is this feature supported?
I've attached the .ae folder

Thanks
Mark
[headrig folder.zip](https://github.com/airbnb/lottie-web/files/3377001/headrig.folder.zip)

**Comment by Danjuanlab** [comment]
Yes, Connector does not seem to be supported yet. The browser complains that velocityAtTime is not a function, but this function is already supported. 
For the same function you can use the joystick and slider, which is currently supported.
